cmd/snap/cmd_run.go

```diff
@@ -99,22 +126,43 @@ func snapRun(snapApp, command string, args []string) error {
 		return fmt.Errorf("cannot find app %q in %q", appName, snapName)
 	}
 
-	// build command to run
+	return runSnapConfine(info, app.SecurityTag(), snapApp, command, args)
+}
+
+func snapRunHook(snapName, hookName, revision string) error {
+	info, err := getSnapInfo(snapName, revision)
+	if err != nil {
+		return err
+	}
+
+	hook := info.Hooks[hookName]
+	if hook == nil {
+		return fmt.Errorf("cannot find hook %q in %q", hookName, snapName)
+	}
+
+	hookBinary := filepath.Join(info.HooksDir(), hook.Name)
+
+	return runSnapConfine(info, hook.SecurityTag(), hookBinary, "", nil)
```

Comment:
Can you please sync with @mvo5 about how we want to get rid of the hookBinary here, per the conversation we've been having around the security of this?  I think @mvo5 is doing this to avoid doing too much at once, which means it's fine to land this as-is, but it'd be good to have the intended behavior there soon considering these issues, which means knowing how to sort it out.